hybris.c

```diff
@@ -1487,6 +1557,84 @@ led_control_htcvision_probe(led_control_t *self)
 }
 
 /* ------------------------------------------------------------------------- *
+ * RGB led control: binary backend
+ * ------------------------------------------------------------------------- */
+
+static void
+led_control_binary_map_color(int r, int g, int b, int *mono)
+{
+  /* Only binary on/off control is available, use
+   * 255 as logical level if nonzero rgb value has
+   * been requested.
+   */
+  *mono = (r || g || b) ? 255 : 0;
+}
+
+static void
+led_control_binary_value_cb(void *data, int r, int g, int b)
+{
+  led_state_binary_t *state = data;
+
+  int mono = 0;
+  led_control_binary_map_color(r, g, b, &mono);
+  led_state_binary_set_value(state + 0, mono);
+}
+
+static void
+led_control_binary_close_cb(void *data)
+{
+  led_state_binary_t *state = data;
+  led_state_binary_close(state + 0);
+}
+
+static bool
+led_control_binary_probe(led_control_t *self)
+{
+  /** Sysfs control paths for RGB leds */
+  static const led_paths_binary_t paths[][1] =
+  {
+    // binary
+    {
+      {
+        .val = "/sys/class/leds/button-backlight/brightness",
```

Comment:
We might want this to be configurable at some point, but probably not needed now.